fe/fe-core/src/main/java/com/starrocks/qe/ConnectProcessor.java

```diff
@@ -548,6 +548,7 @@ private void finalizeCommand() throws IOException {
     }
 
     public TMasterOpResult proxyExecute(TMasterOpRequest request) {
+        ctx.setCurrentCatalog(request.catalog);
```

Comment:
If old version forward to new version, the request.catalog will be null. Will it report NEP?

Reply:
There will no be NPE, but we shall give back a message to tell user upgrade